apps/examples/log_dump/log_dump_main.c

```diff
@@ -58,6 +58,13 @@ int log_dump_main(int argc, char *argv[])
 		goto errout;
 	}
 
+	if (argc > 1) {
+		if (SET_LOGDUMP_FETCH_BLOCKS(fd, argv[1])) {
+			printf("Failed to set the number of blocks to be fetched, errno : %d\n", get_errno());
```

Comment:
Let us add some keywork to indicate that this log is related to log dump. Because, some times lot of logs will be coming. So, if this log does not appear right after entering the command, then its difficult to corelate the error message.